jquery.creditCardValidator.coffee

```diff
@@ -63,7 +63,7 @@ $.fn.validateCreditCard = (callback, options) ->
         }
         {
             name: 'mastercard'
-            pattern: /^5[1-5]/
+            pattern: /^(5[1-5]|2[2-7])/
```

Comment:
The new pattern should be this to cover the new IINs properly:
pattern: /^(5[1-5][0-9]{2}|222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}/